deploy_and_install_scylla_ansible_script/README.md

```diff
@@ -0,0 +1,59 @@
+General info and prerequisites
+==============================
+
+This ansible script will deploy, install and configure ScyllaDB on all supported OS.
+- RHEL7
+- Centos7
+- Ubuntu 14.04
+- Ubuntu 16.04
+- Debian 8
+
+Prerequisites:
+- Ansible 2.3
+- Python 2.7
+
+
+Instructions
+============
+
+This playbook assumes you are installing Scylla using the same disks and NIC for all nodes
+
+
+Running all tasks all host group: ANSIBLE_HOST_KEY_CHECKING=False ansible-playbook scylla_deployment.yml -i servers.ini
+ -t / --tags only runs plays and tasks tagged with these values
+ --skip-tags only runs plays and tasks whose tags do not match these values
+
+
+List all tasks: ansible-playbook scylla_deployment.yml --list-tasks
+
+playbook: scylla_deployment.yml
+ play #1 (host_group_name_from_ini_file): host_group_name_from_ini_file                        TAGS: []
+   tasks:
+     Remove 'abrt' pkg from CentOS / RHEL                                                      TAGS: [1_prereq]
```

Comment:
Why are the tag numbered?
It will be hard to add a step in the middle if required.